test/parallel_exec.jl

```diff
@@ -477,29 +491,6 @@ test_channel(RemoteChannel(()->Channel(10)))
 c=Channel{Int}(1)
 @test_throws MethodError put!(c, "Hello")
 
-c=Channel(256)
-# Test growth of channel
-@test c.szp1 <= 33
-for x in 1:40
-  put!(c, x)
-end
-@test c.szp1 <= 65
-for x in 1:39
-  take!(c)
-end
-for x in 1:64
-  put!(c, x)
-end
-@test (c.szp1 > 65) && (c.szp1 <= 129)
-for x in 1:39
-  take!(c)
-end
-@test fetch(c) == 39
-for x in 1:26
-  take!(c)
-end
-@test isready(c) == false
```

Comment:
In the previous implementation we used to manage a circular buffer which would start small and grow till max size.

In the current implementation, the regular `Array` implementation shrinks and grows the array as required.

The deleted tests were testing growth of the buffer on demand. The user of a `Channel` is not concerned with these internal details.